pkgs/applications/science/math/sage/patches/sphinx-fix-matplotlib-css-perms.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/src/sage_docbuild/ext/multidocs.py b/src/sage_docbuild/ext/multidocs.py
+index f91c7753ca..edeb81ff2e 100644
+--- a/src/sage_docbuild/ext/multidocs.py
++++ b/src/sage_docbuild/ext/multidocs.py
+@@ -284,6 +284,16 @@ def init_subdoc(app):
+         if not app.config.multidoc_first_pass:
+             app.connect('env-updated', fetch_citation)
+ 
++            def fix_matplotlib_css_permissions(app: Sphinx, env):
++                css_file = os.path.join(app.builder.outdir, '_static', 'plot_directive.css')
++                try:
++                    os.chmod(css_file, 0o644)
++                except:
```

Comment:
Note that the above `try`/`except` is just for future-proofing. All `os.chmod` calls introduced by my patch succeed as of Sage 9.6.